### PR TITLE
[feat] Add CLI for containers

### PR DESCRIPTION
**Pull request by mihran113**

### CODE REVIEW

File: src/aimcore/cli/conatiners/commands.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    click.echo('\t'.join(run_hashes))
+    click.echo(f'Total {len(run_hashes)} runs.')
+
+
+@containers.command(name='rm')
+@click.argument('hashes', nargs=-1, type=str)
+@click.pass_context
+@click.option('-y', '--yes', is_flag=True, help='Automatically confirm prompt')
+def remove_runs(ctx, hashes, yes):
```
Comment (alberttorosyan) [inline src/aimcore/cli/conatiners/commands.py:39]: same here; should be remove_containers

File: src/python/aim/_sdk/remote_storage.py [hunk 2]
```diff
@@ -531,4 +531,34 @@ def remove(self, file):
...
+        self.resource_type = 'Repo'
+
+        handler = self._rpc_client.get_resource_handler(self, self.resource_type, args=self.init_args)
+
+        self._resources = RemoteRepoProxy.AutoClean(self)
+        self._resources.rpc_client = client
+        self._resources.handler = handler
+        self._handler = handler
+
+    def _delete_run(self, hash_):
```
Comment (alberttorosyan) [inline src/python/aim/_sdk/remote_storage.py:554]: what about removing the `_`?

**Review by alberttorosyan — COMMENTED** [review]
Left a couple of minor comments (mostly renaming stuff)
One generic thought: with the current implementation we have a bit mixed structure of Repo/ProxyRepo. What about extracting the data management into a separate class which will have it's local/remoteproxy implementations and leave Repo just as a high-level wrapper? 🤔

File: src/python/aim/_sdk/remote_storage.py (hunk 2 above)
Comment (mihran113) [reply src/python/aim/_sdk/remote_storage.py:554]: which `_`?

File: src/aimcore/cli/conatiners/commands.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+def list_runs(ctx):
+    """List Runs available in Repo."""
+    repo_path = ctx.obj['repo']
+    if not Repo.is_remote_path(repo_path):
+        if not Repo.exists(repo_path):
+            click.echo(f'\'{repo_path}\' is not a valid aim repo.')
+            exit(1)
+
+    repo = Repo.from_path(repo_path)
+    run_hashes = repo.container_hashes
```
[the remark this answers is not in the document]
Comment (mihran113) [reply src/aimcore/cli/conatiners/commands.py]: added a `todo` item for this

File: src/python/aim/_sdk/remote_storage.py (hunk 2 above)
Comment (alberttorosyan) [reply src/python/aim/_sdk/remote_storage.py]: the one in the method name. just make those methods a part of public API

**Review by alberttorosyan — APPROVED** [review]
LGTM

File: src/python/aim/_sdk/remote_storage.py (hunk 2 above)
Comment (mihran113) [reply src/python/aim/_sdk/remote_storage.py]: there's already a public method named `delete_run`, this is one is here for the tricks with remote and local

**Pull request merged**